hypothesis-python/src/hypothesis/internal/floats.py

```diff
@@ -19,9 +19,9 @@
 # Format codes for (int, float) sized types, used for byte-wise casts.
 # See https://docs.python.org/3/library/struct.html#format-characters
 STRUCT_FORMATS = {
-    16: (b"!H", b"!e"),  # Note: 'e' is new in Python 3.6, so we have helpers
-    32: (b"!I", b"!f"),
-    64: (b"!Q", b"!d"),
+    16: ("!H", "!e"),  # Note: 'e' is new in Python 3.6, so we have helpers
```

Comment:
there shouldn't be helpers now that py3.5 support is dropped

Reply:
And in fact I cleaned up the helpers but missed the comment :facepalm: 

https://github.com/graingert/hypothesis/commit/4e94c526e9b6cc50d00fbb222b3574546d64b14f#diff-8e2cdbf7b8a84f330d76ac3f7fb8bbd5